Makefile

```diff
@@ -83,6 +87,10 @@ cluster-sync: kustomize
 cluster-sync-containerdisks:
 	scripts/kubevirtci.sh sync-containerdisks
 
+.PHONY: cluster-sync-ol-containerdisks
+cluster-sync-ol-containerdisks:
```

Comment:
Can you fold this under cluster-sync-containerdisks in both cases?

Reply:
Sorry, I'm not sure if I'm getting it correctly. Are you suggesting something like this?
```makefile
.PHONY: cluster-sync-containerdisks
cluster-sync-containerdisks:
	scripts/kubevirtci.sh sync-containerdisks
        scripts/kubevirtci.sh sync-ol-containerdisks
```